suite-native/transactions/src/components/TransactionsList/TransactionList.tsx

```diff
@@ -238,9 +238,15 @@ export const TransactionList = ({
 
     const renderItem = useCallback(
         ({ item, index }: { item: TransactionListItem; index: number }) => {
+            // month with only month name and without token txn
+            const isEmptyMonth = typeof data.at(index + 1) === 'string' || !data.at(index + 1);
```

Comment:
Could you please move this const inside of the `if (typeof item === 'string') {` block?

Because it seems that:
- a) the value is not used anywhere else
- b) if the `if` condition is not truthy, then the `isEmptyMonth` could be true for the last transaction item of the month and then the naming does not say the truth. Because it will get `true` for not empty month. Of course it works the way you have it now, but it is bit confusing to me. 

What do you think? :)

Reply:
definitely! 👍 
c10c018604